assets/src/js/batch.jsx

```diff
@@ -236,6 +245,12 @@ var App = React.createClass( {
 					batchErrors={ this.state.errors }
 					toggleProcessing={ this.toggleProcessing }
 				/>
+
+				<ModalError
```

Comment:
Ah ok. IMO, for clarity, we should either rename the file + component to `ModalError` or change the import + usage to `ModalReset` to match the file. The current mix of `ModalError` and `ModalReset` is confusing. Unless I'm just missing something 🤔